React/Modules/RCTStatusBarManager.m

```diff
@@ -52,9 +83,12 @@ static BOOL RCTViewControllerBasedStatusBarAppearance()
 
 - (void)startObserving
 {
+#if !TARGET_OS_TV
```

Comment:
Same here: it seems it would be safer to just gate this whole module. tvOS doesn't have a status bar.